src/vm/mod.rs

```diff
@@ -234,13 +234,14 @@ pub fn is_reserved(name: &str) -> bool {
     }
 }
 
-/* This function evaluates a list of expressions, sharing a global context.
- * It returns the final evaluated result.
- */
+/// This function evaluates a list of expressions, sharing a global context.
+/// It returns the final evaluated result.!
+/// Used for the initialization of a new contract.
```

Comment:
nit: if you're refactoring to doc comments, perhaps use `/** ... */` instead of `///`, since this is a multiline comment

Reply:
I was attempting to follow the style guide here: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text 

Also there appears to be a lot of uses of the triple slash docs comment in our repo - lmk if I should change it anyway.